### DISCUSSION TITLE
Small and simple example to change the texture basemap of a model?

### DISCUSSION THREAD

**Discussion by wox76** [google/model-viewer#3988] [General]
Small and simple example to change the texture base of a model?
I've tried and tried but I couldn't do it.
In Normal yes, but in BaseMap no! :(
I simply have to load a model and assign a texture I decide on to a certain material.
Thank you all!

**Comment by elalish** [comment]
Have you followed our [example](https://modelviewer.dev/examples/scenegraph/#swapTextures)? An easy mistake to make is the difference between `material.normalTexture` and `material.pbrMetallicRoughness.baseColorTexture`.

**Reply by wox76** [reply]
Yes but I didn't quite understand :(
I just need to figure out how to do the basecolor textures swap.
